app/uk/gov/hmrc/tradergoodsprofiles/connectors/RouterConnector.scala

```diff
@@ -87,13 +87,12 @@ class RouterConnector @Inject() (
 
   def removeRecord(eori: String, recordId: String, actorId: String)(implicit hc: HeaderCarrier): Future[HttpResponse] =
     withMetricsTimerAsync("tgp.removerecord.connector") { _ =>
-      val url = appConfig.routerUrl.withPath(routerRoute(eori, recordId))
+      val url = appConfig.routerUrl.toUrl + routerRouteRemoveRecord(eori, recordId, actorId)
 
       httpClient
-        .put(url"$url")
+        .delete(url"$url")(hc)
```

Comment:
you do not need to add hc here as this is implicit.

Reply:
> you do not need to add hc here as this is implicit.

Removed